public/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Hiba's Angels Autocomplete</title>
+    <link rel="stylesheet" href="style.css">
```

Comment:
do you want to put href="./style.css" ? This is better in my opinion because it's more explicit